src/components/main/person-request/PersonRequestDetail.vue

```diff
@@ -0,0 +1,104 @@
+<script>
+  import personRequestService from 'services/person-request';
+
+  export default {
+
+    data() {
+      return {
+        isLoading: false,
+        activeTab: '1',
+        personRequest: {}
+      };
+    },
+
+    mounted() {
+      this.isLoading = true;
+      this.getPersonRequest(this.$route.params.personRequestId);
+    },
+    computed: {
+      position() {
+        const position = {
+          lat: 0,
+          lng: 0
+        };
+        if (!this.personRequest.geo) { return position; }
+
+        position.lat = this.personRequest.geo.loc[1];
+        position.lng = this.personRequest.geo.loc[0];
+        return position;
+      }
+    },
+    methods: {
+      onSelect(personRequest) {
+        // this.$router.push({
+        //   name  : 'person-request-detail',
+        //   params: { personId: personRequest._id }
+        // });
+      },
+      getPersonRequest(personRequestId) {
+        personRequestService.getById(personRequestId)
+          .then((req) => {
+            this.personRequest = req;
+            console.log(this.personRequest);
+            this.isLoading     = false;
+          });
+      },
+      deletePersonRequest(index, personRequest) {
```

Comment:
Do we have delete functionality on person request detail? If don't, lets remove this logic.